### PR TITLE
1822: Initial commit with base config and basic engine

### PR DESCRIPTION
**Pull request by magnusfalt**
* Most of the map is done, London and minor 14 need some work.
* All the privates, minors, majors, tiles, trains and phases are in the config but with no abilities or events.
* Some basic logos and icons.

### CODE REVIEW

File: lib/engine/config/game/g_1822.rb [hunk 1]
```diff
@@ -0,0 +1,1647 @@
...
+    },
+    "X19": {
+      "count": 4,
+      "color": "gray",
+      "code": "city=revenue:50,slots:3;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;path=a:4,b:_0;path=a:5,b:_0"
+    }
+  },
+  "market": [
+    [
+      "",
```
Comment (perwestling) [inline lib/engine/config/game/g_1822.rb:273]: You can remove the empty strings - the code handles this.

File: lib/engine/config/game/g_1822.rb [hunk 2]
```diff
@@ -0,0 +1,1647 @@
...
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "H33",
+      "color": "swrGray",
+      "text_color": "black"
+    }
+  ],
+  "trains": [
+    {
+      "name": "L",
```
Comment (perwestling) [inline lib/engine/config/game/g_1822.rb:1109]: I am not sure if the code supports the L train.

**Review by perwestling — COMMENTED** [review]
Impressive commit!

File: lib/engine/config/game/g_1822.rb (hunk 2 above)
Comment (magnusfalt) [reply lib/engine/config/game/g_1822.rb:1109]: No i just wanted as much as i can in the config. I will start to handle the accual code now.

Comment (jenf) [reply lib/engine/config/game/g_1822.rb:1109]: Probably not, but i'd suggest a separate PR for this

Comment (magnusfalt) [reply lib/engine/config/game/g_1822.rb:1109]: I figured it doesnt matter if it doesnt work at the moment as nothing else works as it should. But i can remove the L train along with the phase 1.

**Comment by tobymao** [comment]
can you compress the svgs there's a guide in the wiki

**Review by jenf — APPROVED** [review]

**Pull request merged**